go.mod

```diff
@@ -6,4 +6,16 @@ module github.com/trustbloc/edge-core
 
 go 1.13
 
-require github.com/stretchr/testify v1.4.0
+require (
+	github.com/flimzy/diff v0.1.6 // indirect
+	github.com/flimzy/testy v0.1.16 // indirect
+	github.com/go-kivik/couchdb v2.0.0+incompatible
```

Comment:
why it's incompatible

Reply:
I don't think we have a choice. The go-kivik module has the same and (i believe) go mod tidy will put it back if you try to take it out.

Reply:
@fqutishat The "incompatible" thing has something to do with the way the go modules are set up in the Go Kivik library. I don't think I can do anything about it until they update it. According to what I've read it shouldn't have any impact on us though. See https://stackoverflow.com/questions/57355929/what-incompatible-in-go-mod-mean-will-it-make-harm for more info

The author of the Go Kivik repo has an open issue for go.mod support, so I think when that's resolved then this "incompatible" thing will go away (after we update the dependency of course): https://github.com/go-kivik/kivik/issues/417